packages/eslint-config-4catalyzer-typescript/rules.js

```diff
@@ -39,6 +39,20 @@ module.exports = {
   // Match something like a stricter upstream camelcase.
   '@typescript-eslint/naming-convention': [
     'error',
+    {
```

Comment:
Are these rules evaluated in order and short circuited? Assuming so, thus putting this first

Reply:
no idea!

Reply:
OK, it looks like it as the existing rules for this are seemingly mutually exclusive.